pkgs/applications/gis/grass/default.nix

```diff
@@ -6,24 +6,33 @@
 
 stdenv.mkDerivation rec {
   pname = "grass";
-  version = "8.2.0";
+  version = "8.2.1";
 
   src = with lib; fetchFromGitHub {
     owner = "OSGeo";
     repo = "grass";
     rev = version;
-    sha256 = "sha256-VK9FCqIwHGmeJe5lk12lpAGcsC1aPRBiI+XjACXjDd4=";
+    sha256 = "sha256:02fxcpdwkjis0i5mqks1by4kwazi1947flmhfd0yz2xxgdvx0wsk";
   };
 
   nativeBuildInputs = [
     pkg-config bison flex makeWrapper wrapGAppsHook
-    gdal geos libmysqlclient netcdf pdal
+    gdal # for `gdal-config`
+    geos # for `geos-config`
+    netcdf # for `nc-config`
+    libmysqlclient # for `mysql_config`
+    pdal # for `pdal-config`; remove with next version, see https://github.com/NixOS/nixpkgs/pull/215960#issuecomment-1568854089
   ] ++ (with python3Packages; [ python-dateutil numpy wxPython_4_2 ]);
```

Comment:
Changes in `nativeBuildInputs` look ok, the only thing I suggest to change is comment on `pdal` line. I suggest to point it directly to the [GRASS PR](https://github.com/OSGeo/grass/pull/2851) where they made change from `pdal-config` to `pkg-config`.

Reply:
Done.